README.md

```diff
@@ -522,37 +563,60 @@
     if (test)
       return false;
 
+    // acceptable (only for very simple statements and early function return)
+    if (test) return false;
+
     // good
     if (test) {
       return false;
     }
 
-    // good
-    function () { return false; }
+    // bad
```

Comment:
lol, you are correct sir, using without curly braces most certainly does not work with functions. I didn't notice we had switched from `if` to `function`. OK, well, I can't even remember the last time I wrote a one line function that returned false, so, I guess I don't care either way.